data/sql/updates/pending_db_world/rev_1680525460717061700.sql

```diff
@@ -0,0 +1,3 @@
+DELETE FROM `conditions` WHERE `SourceTypeOrReferenceId` = 1 AND `SourceGroup` = 17798 AND `SourceEntry` = 24313 AND `ConditionTypeOrReference` = 7;
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`,`ConditionTypeOrReference`, `ConditionValue1`, `ConditionValue2`) VALUES
+(1, 17798, 24313, 7, 197, 1);
```

Comment:
Oh, oops, I didn't notice earlier but it's missing comment

Reply:
I skipped the comment field because there is no row in conditions with a comment